packages/dashboard/src/components/app-events.ts

```diff
@@ -9,4 +16,6 @@ export const AppEvents = {
   robotSelect: new Subject<[fleetName: string, robotName: string] | null>(),
   taskSelect: new Subject<TaskState | null>(),
   refreshTaskQueueTableCount: new Subject<number>(),
+  refreshAlertCount: new Subject<number>(),
+  alertListOpenedAlert: new Subject<Alert | null>(),
```

Comment:
Yup it is intentional for readability, since a `list` could be many things (from robot list or task list, etc), the alert list would refer to the menu list explicitly